recipes/ncfp/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
+
+# Add more build steps here, if they are necessary.
```

Comment:
@widdowquinn can you please clean up the recipe a little bit by removing the comments.